tests/e2e/test_rosacli_cluster_post.go

```diff
@@ -237,3 +238,52 @@ var _ = Describe("Healthy check",
 				Expect(etcdEncryption).To(BeTrue())
 			})
 	})
+
+var _ = Describe("Create cluster with the version in some channel group testing",
+	labels.Feature.Cluster,
+	func() {
+		defer GinkgoRecover()
+		var (
+			clusterID      string
+			rosaClient     *rosacli.Client
+			clusterService rosacli.ClusterService
+		)
+
+		BeforeEach(func() {
+			By("Get the cluster")
+			var clusterDetail *profilehandler.ClusterDetail
+			var err error
+			clusterDetail, err = profilehandler.ParserClusterDetail()
+			Expect(err).ToNot(HaveOccurred())
+			clusterID = clusterDetail.ClusterID
+			Expect(clusterID).ToNot(Equal(""), "ClusterID is required. Please export CLUSTER_ID")
+
+			By("Init the client")
+			rosaClient = rosacli.NewClient()
+			clusterService = rosaClient.Cluster
+		})
+
+		AfterEach(func() {
+			By("Clean remaining resources")
+			rosaClient.CleanResources(clusterID)
+		})
+
+		It("User can create cluster with channel group - [id:35420]",
+			labels.Medium, labels.Runtime.Day1Post,
+			func() {
+				clusterName := "ocp-35420"
+				profile := profilehandler.LoadProfileYamlFileByENV()
+
+				By("Check if the cluster using right channel group")
+				versionOutput, err := clusterService.GetClusterVersion(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(versionOutput.ChannelGroup).To(Equal(profile.ChannelGroup))
+
+				By("Create cluster with version not in channel group")
```

Comment:
Split the negative checkpoint to another case with day1-negative labeled

Reply:
create a new test cases ocp-74399 and split the checkpoint to the new case with day1-negtaive label.